### PR TITLE
feature: added support for china region

### PR DESCRIPTION
**Pull request by james03160927**
## Description

Fixes https://github.com/gruntwork-io/cloud-nuke/issues/126

## TODOs

Read the [Gruntwork contribution guidelines](https://gruntwork.notion.site/Gruntwork-Coding-Methodology-02fdcd6e4b004e818553684760bf691e).

- [ ] Update the docs.
- [ ] Run the relevant tests successfully, including pre-commit checks.
- [ ] Ensure any 3rd party code adheres with our [license policy](https://www.notion.so/gruntwork/Gruntwork-licenses-and-open-source-usage-policy-f7dece1f780341c7b69c1763f22b1378) or delete this line if its not applicable.
- [ ] Include release notes. If this PR is backward incompatible, include a migration guide.
- [ ] Attention Grunts - if this PR adds support for a new resource, ensure the `nuke_sandbox` and `nuke_phxdevops` jobs in `.circleci/config.yml` have been updated with appropriate exclusions (either directly in the job or via the `.circleci/nuke_config.yml` file) to prevent nuking IAM roles, groups, resources, etc that are important for the test accounts.

## Release Notes (draft)

Added / Removed / Updated [X].

### CODE REVIEW

File: aws/region.go [hunk 1]
```diff
@@ -151,25 +148,3 @@ func GetTargetRegions(enabledRegions []string, selectedRegions []string, exclude
 	}
 	return targetRegions, nil
 }
-
-func Session2cfg(ctx context.Context, session *session.Session) (awsgoV2.Config, error) {
```
Comment (wakeful) [inline aws/region.go:155]: can you rebase?

Comment (james03160927) [reply aws/region.go]: Should be good now. Thanks.

**Comment by denis256** [comment]
Looks like changes are failing to compile

```
# github.com/gruntwork-io/cloud-nuke/aws/resources
aws/resources/ec2_placement_group.go:32:16: configObj.EC2PlacementGroups undefined (type config.Config has no field or method EC2PlacementGroups)
aws/resources/ec2_placement_group_types.go:41:19: configObj.EC2PlacementGroups undefined (type config.Config has no field or method EC2PlacementGroups)
```

**Comment by james03160927** [comment]
Rebased again. Should be good now.

**Review by gcagle3 — APPROVED** [review]

**Pull request merged**